src/routes/embedded/auth/recover-account/select-account/auth-recover-account-select.view.tsx

```diff
@@ -0,0 +1,54 @@
+import { useEmbedded } from "~utils/embedded/embedded.hooks";
```

Comment:
I think we might need to update this screen also with better UI.

Reply:
Agree. Those UUID don't let me know which social account or login method I'll be recovering